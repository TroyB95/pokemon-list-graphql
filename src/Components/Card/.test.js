import React from "react";
import { shallow } from "enzyme";

import CardComponent from "./index";

describe("CardComponent", () => {
  it("should render an image, name and number", () => {
    const component = shallow(<CardComponent cardImg="" cardName="Test Name" cardNumber="001" />);

    expect(component).toMatchSnapshot();
  });
});
