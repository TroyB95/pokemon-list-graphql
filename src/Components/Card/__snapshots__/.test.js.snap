// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardComponent should render an image, name and number 1`] = `
<div
  className="card"
>
  <img
    alt="the pokemon Test Name"
    className="card__image"
    src=""
  />
  <p
    className="card__name"
  >
    Test Name
  </p>
  <p
    className="card__number"
  >
    001
  </p>
</div>
`;
